sunkit_image/coalignment_module/__init__.py

```diff
@@ -0,0 +1,5 @@
+from sunkit_image.coalignment_module.interface import coalignment
+from sunkit_image.coalignment_module.match_template import match_template_coalign
+from sunkit_image.coalignment_module.util.decorators import register_coalignment_method
+
+__all__ = ["coalignment", "match_template_coalign", "register_coalignment_method"]
```

Comment:
The top level api should just be the main API and the register method.

Reply:
I moved the registered method decorator in the already existing utils. Also the import for the match_template_coalign needs to stay so that it gets registered in the global dictionary.

Reply:
I'll have to look again but I don't see why the method needs to in the top level init to register it.

Reply:
> I'll have to look again but I don't see why the method needs to in the top level init to register it.

it just needs to be imported once, either by the user or by default for the decorator to register it.